vsts/scripts/release-runtimeimages.sh

```diff
@@ -11,6 +11,6 @@ while read sourceImage; do
   docker pull "$sourceImage"
   newtag=$(echo "$sourceImage" | sed -r 's/oryxdevms/oryxprod/')
   echo "$newtag"
-  docker tag "$sourceImage" "$newtag"
-  docker push "$newtag"
-done <"$(System.DefaultWorkingDirectory)/_Oryx-Runtime/drop/images/runtime-images.txt"
+  docker tag "$sourceImage" "$newtag"``
```

Comment:
I checked in release we don't use this script, can you make sure we use this script to release runtime images https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-releaseManagement-web.cd-workflow-hub?definitionId=1380&_a=definition-tasks&environmentId=6856